Wrappers/TextBox.cs

```diff
@@ -0,0 +1,43 @@
+﻿using OpenQA.Selenium;
+using SeleniumExtras.WaitHelpers;
+using Utilities;
+
+namespace Wrappers
+{
+    public class TextBox : WebPageElement
+    {
+        public TextBox(IWebElement element) : base(element)
+        {
+        }
+
+        public TextBox(By locator) : base(locator)
+        {
+        }
+
+        public void ClearAndEnterText(string text)
+        {
+            Element.Clear();
+            Element.SendKeys(text);
+        }
+
+        public void DeleteAllTextWithKey()
+        {
+            Element.SendKeys(Keys.Control + "a");
+            Element.SendKeys(Keys.Backspace);
+        }
+
+        private void DeleteAndEnterText(string text, By locator)
```

Comment:
our Element should be immutable, but here we not follow the rule, please fix it
Also please lets make our Element field readonly and we will not have opportunity to do it

Reply:
OK, done